src/App.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 import { Switch, Route } from 'react-router';
+import { Store } from 'redux-starter-kit';
+import { Provider as ReduxProvider } from 'react-redux';
+import { OidcProvider, loadUser } from 'redux-oidc';
 
+import userManager from './oidc/userManager';
+import enableOidcLogging from './oidc/enableOidcLogging';
 import Home from './pages/Home';
+import OidcCallback from './pages/OidcCallback';
 
-const App: React.FC = () => {
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Is this somehow temporary?

Reply:
oidc-client does not log anything by default, I just enabled logging (to console) that could help when developing (e.g. user logged in, token expired.. might be helpful)